app/controllers/state_file/questions/az_dependents_dob_controller.rb

```diff
@@ -0,0 +1,35 @@
+module StateFile
+  module Questions
+    class AzDependentsDobController < QuestionsController
+      layout "state_file/question"
+
+      def edit
+        @intake = current_intake
+        @dependents = @intake.dependents
+        @form = form_class.from_intake(@intake)
```

Comment:
i think you can delete this whole override. you should have access to `current_intake` in the view, which would also give you access to dependents, and the form instantiation here appears to be identical to what's in questions_controller